ff_time.c

```diff
@@ -105,8 +104,8 @@
     #define WEEK_DAY_SATURDAY       6
 
 /* Make a bitmask with a '1' for each 31-day month. */
-    #define _MM( month )    ( 1u << ( month - 1 ) )
-    #define MASK_LONG_MONTHS     ( _MM( 1 ) | _MM( 3 ) | _MM( 5 ) | _MM( 7 ) | _MM( 8 ) | _MM( 10 ) | _MM( 12 ) )
+    #define MM_( month )    ( 1u << ( month - 1 ) )
```

Comment:
Just curious: why this change: from `_MM` to `MM_`? Did you run MISRA checks?

Reply:
@htibosch - macros and precompiler definitions that start with `_` are considered reserved and should not be used.
https://clang.llvm.org/docs/DiagnosticsReference.html#wreserved-identifier

For MISRA checks - if they were run in the ci.yml then they pass.  If not then I suggest if this is a requirement then you should add it to the ci.yml to check that.